packages/react-core/src/components/Tabs/examples/Tabs.md

```diff
@@ -2,7 +2,7 @@
 id: Tabs
 section: components
 cssPrefix: pf-c-tabs
-propComponents: ['Tabs', 'Tab', 'TabContent', 'TabTitleText', 'TabTitleIcon']
+propComponents: ['Tabs', 'Tab', 'TabContent', 'TabTitleText', 'TabTitleIcon', 'OverflowTab']
```

Comment:
We should only export the OverflowTab if we have an example which imports it. We can always export it later, but making 'breaking changes' is much easier if it's never exported. We should err on the side of only exporting things we know the consumer will need.